src/components/Introduction.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <div>
+    <h1>Introduction </h1>
+    <p> NOTE: Styling is incoming! </p>
+    <p> 
```

Comment:
So I think I should have mentioned this when we spoke about the task sorry but I was thinking of something along the lines of the image I've attached. Where we grab the users attention with a headline and some supporting text.
Hope this makes sense?

![image](https://user-images.githubusercontent.com/46312751/118522405-4e080000-b73c-11eb-83cc-8c35036fd314.png)